app/experimenter/kinto/tasks/nimbus.py

```diff
@@ -0,0 +1,164 @@
+import markus
+from celery.utils.log import get_task_logger
+from django.conf import settings
+from django.contrib.auth import get_user_model
+
+from experimenter.celery import app
+from experimenter.experiments.api.v6.serializers import NimbusExperimentSerializer
+from experimenter.experiments.changelog_utils import generate_nimbus_changelog
+from experimenter.experiments.models import (
+    NimbusBucketRange,
+    NimbusExperiment,
+    NimbusIsolationGroup,
+)
+from experimenter.kinto.client import KintoClient
+
+logger = get_task_logger(__name__)
+metrics = markus.get_metrics("kinto.tasks")
+
+
+def get_kinto_user():
+    user, _ = get_user_model().objects.get_or_create(
+        email=settings.KINTO_DEFAULT_CHANGELOG_USER,
+        username=settings.KINTO_DEFAULT_CHANGELOG_USER,
+    )
+    return user
+
+
+@app.task
+@metrics.timer_decorator("push_experiment_to_kinto.timing")
+def nimbus_push_experiment_to_kinto(experiment_id):
+    kinto_client = KintoClient(settings.KINTO_COLLECTION_NIMBUS)
+
+    metrics.incr("push_experiment_to_kinto.started")
+
+    experiment = NimbusExperiment.objects.get(id=experiment_id)
+    if not NimbusBucketRange.objects.filter(experiment=experiment).exists():
+        NimbusIsolationGroup.request_isolation_group_buckets(
+            experiment.slug,
+            experiment,
+            NimbusExperiment.BUCKET_COUNT,
+        )
+
+    data = NimbusExperimentSerializer(experiment).data
```

Comment:
This could never fail to serialize? Or is the default error handling sufficient for this case?

Reply:
Yeah if we wind up in a situation where an experiment makes it here and completely fails to serialize then we should just blow up, abort, and report the error.